crates/sequencer/src/commitment_controller.rs

```diff
@@ -46,15 +46,31 @@ pub fn get_commitment_info(
     // if there is a height then start from height + 1 and go to prev_l1_height
     let (l2_range_to_submit, l1_height_range) = match last_commitment_l1_height {
         Some(last_commitment_l1_height) => {
-            let l1_height_range = (last_commitment_l1_height.0 + 1, prev_l1_height);
+            let l1_start = last_commitment_l1_height.0;
```

Comment:
set l1_start = last_commitment_l1_height.0 + 1

for easier understandability in the future, also below works incorrectly bc of this :D

Reply:
then line 65

let l1_height_range = (l1_start + 1, l1_end);


becomes

let l1_height_range = (l1_start, l1_end);